DOCS/man/vf.rst

```diff
@@ -247,12 +247,13 @@ Available mpv-only filters are:
         :bt.601-525:   ITU-R BT.601 (SD) 525-line systems (NTSC, SMPTE-C)
         :bt.601-625:   ITU-R BT.601 (SD) 625-line systems (PAL, SECAM)
         :bt.709:       ITU-R BT.709 (HD) (same primaries as sRGB)
-        :bt.2020:      ITU-R BT.2020 (UHD)
+        :bt.2020:      ITU-R BT.2020 (UHD), default for HDR transfers
         :apple:        Apple RGB
         :adobe:        Adobe RGB (1998)
         :prophoto:     ProPhoto RGB (ROMM)
         :cie1931:      CIE 1931 RGB
         :dci-p3:       DCI-P3 (Digital Cinema)
+        :display-p3:   P3-D65 (all modern Apple, use with PQ by Netflix masters)
```

Comment:
See above, this seems unnecessary and goes against the convention of the rest of the docs